### DISCUSSION TITLE
Transição para Front End

### DISCUSSION THREAD

**Discussion by felipem1511** [frontendbr/forum#416] [Discussão]
Then everybody,

Vou fazer um breve resumo sobre essa issue. Sou formado em Gestão em TI, porém no meu primeiro emprego na área, _tirando um estágio em redes_, entrei na área de modelagem de processos para aplicativos, pois não tinha ninguém que o fizesse. Desenvolvia protótipos e me aventurava um pouco em infra, especificamente barramento Oracle e WSO2. Mas continuava com os processos BPM.

Hoje trabalho como Consultor de Negócios, mas me descobri um apaixonado por Front End sem conhecer o "título". 

Desenvolvo a 3 anos alguns projetos Android com uns amigos e neste período sempre me dividindo entre negócios e dev.

Pois bem, comecei um processo de transição de carreira em agosto de 2016, fazendo vários cursos para lembrar de algumas coisas e fixar outras, como: **HTML, CSS e JS**. Agora estou partindo para **Gulp, Grunt, React, Sass e Regex**.

Gostaria de algumas dicas, tutoriais, materiais, blogs, e o que mais puderem me ajudar. 

Tenho uma meta de trabalhar com Front End até o final do ano.

Help me folks, estou no caminho certo?

PS: Tenho 25 anos

**Comment by matheusml** [comment]
Oi @felipem1511 

Bom saber que você está vindo pro FE!

Se você quer tutoriais sobre React, eu indico o [JSCasts](http://jscasts.teachable.com/).
E sobre material sobre JavaScript, você poder acompanhar [essa issue](https://github.com/frontendbr/forum/issues/410).

Abraços!

**Comment by douglas-castilho** [comment]
mais uma referência 
https://github.com/training-center/mentoria

**Comment by felipem1511** [comment]
Que beleza galera, muito obrigado. Vocês acreditam que com o conhecimento que tenho eu conseguiria alguma vaga de Jr?

Experiência: 

- HTML

- CSS

- JS

- SaSS

- SOA

- REST

**Comment by theandersonn** [comment]
@felipem1511 segue duas indicações de cursos [são pagos], possuem cursos rápidos que nos ajudam a ter entendimento de assunto ou start para evoluirmos gradualmente.

- https://www.alura.com.br/
- https://www.schoolofnet.com/

Força ai na jornada! Sim, você com o que você mencionou consegue uma vaga. Marque presença no GitHub com projetos legais de coisas que estiver estudando no momento, conta bastante numa entrevista.

Desencana com a idade, ok?

Eu comecei com 33 ;)

**Comment by felipem1511** [comment]
@theandersonn muito obrigado pelo apoio. Estou fazendo vários cursos da Alura já. Eu já os conhecia e estou refazendo alguns cursos e aprendendo novos. A school of net Vou olhar com mais calma.

Muito obrigado pessoal

**Comment by guilhermejcgois** [comment]
> Vocês acreditam que com o conhecimento que tenho eu conseguiria alguma vaga de Jr?
> 
> Experiência:
> 
>     HTML
>     CSS
>     JS
>     SaSS
>     SOA
>     REST
> 

Cara, acredito que sim viu, esse ano comecei como Jr. em uma startup aqui em São Paulo, não pediram nada de pré-processadores e _web services_ como pré-requisitos, o mais punk foi Angular 2 (que tive meu primeiro contato com o teste que me passaram no processo) que era um desejável, hoje estou aprendendo (e apanhando rsrs) e aplicando no trabalho, acho que vai da empresa, mas é possível sim achar uma vaga de júnior com esses requisitos!

No meu caso como eu já tinha brincado um pouco com Gulp, ReactJS e SaSS no meu estágio, acredito que isso me ajudou.

Abraços e sucesso nessa nova empreitada! o/

**Comment by felipem1511** [comment]
@guilhermejcgois Eu já mexi um pouco com o Angular 1, mas nem coloco como experiência kkkk. Estou na procura de uma vaga... Enquanto isso vou me desenvolvendo e acompanhando as issues do pessoal aqui.

Algo que reparei também, foi que aqui em Brasília é bem complicado uma vaga de Jr, mas já estou disposto a me mudar para outro estado, caso surja a oportunidade.

Valeu pela ajuda e sucesso na Startup!  \o

**Comment by guilhermejcgois** [comment]
@felipem1511 opa, tamo ai pra isso ;) Valeu o/

Já que está a procura de vaga, em dezembro do ano passado vi o @felquis compartilhando uma vaga da Pagar.me aqui em São Paulo, se tiver interesse entra em contato com ele, mas não sei se já não fecharam a vaga...
